examples/sdk/reactNative/android/app/build.gradle

```diff
@@ -47,7 +49,7 @@ react {
     // hermesCommand = "$rootDir/my-custom-hermesc/bin/hermesc"
     //
     //   The list of flags to pass to the Hermes compiler. By default is "-O", "-output-source-map"
-    // hermesFlags = ["-O", "-output-source-map"]
+    hermesFlags = ["-O", "-output-source-map"]
```

Comment:
if this is the default option, does this have to be uncommented?

Reply:
I left it uncommented because our example app should always show all benefits from our SDK. I think we want this option to be enabled by default.